lk2nd/device/dts/msm8916/msm8916-samsung.dts

```diff
@@ -841,4 +841,23 @@
 			};
 		};
 	};
+	
+	grandmaxltezc {
+		model = "Samsung Galaxy Grand Max (SM-G7200)";
```

Comment:
Put this after comments, `/* rev 8 */`, https://github.com/msm8916-mainline/lk2nd/blob/main/lk2nd/device/dts/msm8916/msm8916-samsung.dts#L717

Reply:
It's still not sorted in revision order. There are 11 revisions, and 8 < 11.